zero/src/bin/trie_diff.rs

```diff
@@ -0,0 +1,172 @@
+//! This binary is a debugging tool used to compare
```

Comment:
nit: I'd move this to `Cli`, so it shows up with `--help`

Reply:
Will do in the next PR on this topic, I am currently working on.